src/WebJobs.Extensions.DurableTask/Auth/AzureIdentityOptions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Auth
+{
+    /// <summary>
+    /// Represents the identity of an Azure service, application, or user.
+    /// </summary>
+    public class AzureIdentityOptions
+    {
+        /// <summary>
+        /// Gets or sets the identifier for the Azure Active Directory (AAD) tenant.
+        /// </summary>
+        public string TenantId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the identifier for the application (client).
+        /// </summary>
+        public string ClientId { get; set; }
+
+        /// <summary>
+        /// Gets or sets the application client's secret.
+        /// </summary>
+        public string ClientSecret { get; set; }
+
+        /// <summary>
+        /// Gets or sets a moniker indicating the type of authentication to be used.
+        /// </summary>
+        public string Credential { get; set; }
```

Comment:
The only reason is because this appears to be the value used by `Azure.Identity` and in turn the WebJobs SDK: https://github.com/Azure/azure-sdk-for-net/blob/9c9ec82c36580deff41b8940af420519a28cd4e1/sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ClientFactory.cs#L88